ui/apps/platform/cypress/integration/dashboard.test.js

```diff
@@ -132,24 +137,23 @@ describe('Dashboard page', () => {
     });
 
     it('should display top risky deployments', () => {
-        cy.intercept('GET', api.risks.riskyDeployments, {
-            fixture: 'risks/riskyDeployments.json',
-        }).as('riskyDeployments');
+        const { table } = baseSelectors;
 
-        cy.visit(dashboardUrl);
-        cy.wait('@riskyDeployments');
-
-        cy.get(selectors.sectionHeaders.topRiskyDeployments).next().as('list');
-
-        // When the API is not mocked, the UI requests the top 5 deployments from the server.
-        // Here we are intercepting the call and injecting 6 deployments, so that is what
-        // the component will display.
-        cy.get('@list').find('li').should('have.length', 6);
-
-        cy.get(selectors.buttons.viewAll).click();
-        cy.url().should('match', /\/main\/risk/);
+        visitMainDashboard();
 
-        // TODO: validate clicking on any sector (for some reason '.click()' isn't stable for D3 chart)
+        // Gets the list of top risky deployments on the dashboard and checks to
+        // see that the order matches on the Risk page.
+        cy.get(`${selectors.sectionHeaders.topRiskyDeployments} + div li`).then(($deployments) => {
+            cy.get(selectors.buttons.viewAll).click();
+            cy.url().should('match', /\/main\/risk/);
```

Comment:
For consistency with other similar assertions, can rewrite as `cy.location('pathname').should('eq', riskUrl);` and then add

```js
import { url as riskUrl } from '../constants/RiskPage';
```